relay-wstring/src/lib.rs

```diff
@@ -0,0 +1,512 @@
+//! A UTF-16 little-endian string type.
+//!
+//! The main type in this crate is [WStr] which is a type similar to [str] but with the
```

Comment:
Since we still need to generate docs with stable, can you please replace all these with proper links and quoting? Currently, it will render like this on our docs page:

![image](https://user-images.githubusercontent.com/1433023/93602116-77606880-f9c2-11ea-97d2-096f0cdf676f.png)